packages/trader/src/sass/app/_common/components/contract-type-info.scss

```diff
@@ -26,6 +26,7 @@
         }
     }
     &__content {
+        width: 98%;
```

Comment:
@henry-deriv in mobile it looks like it includes only the video. please check 
![Uploading Screenshot 2023-10-26 at 10.13.36 AM.png…]()